src/components/cards/SponsorCard.tsx

```diff
@@ -48,20 +51,27 @@ export default function SponsorCard({ sponsor }: SponsorCardProps) {
           />
         </View>
         <View style={CardStyles.headerTitle}>
-          <Text style={headingStyle}>{sponsor.sponsorName}</Text>
-          <Text style={primaryTextStyle}>{sponsor.email}</Text>
+          <Text style={headingStyle}>{sponsor.sponsor_name}</Text>
         </View>
       </View>
       <View
         style={[
           CardStyles.tierBadge,
-          { backgroundColor: tierColor(sponsor.subscriptionTier) },
+          {
+            backgroundColor: tierColor(
+              sponsor.subscription_tier.includes('Partner')
```

Comment:
Are you accounting for partner specially because you don't expect it to be in the db?

Reply:
Its stored as "Partner Organization" so we're cutting it to "Partner" since it takes up less space. It was a change we decided to make after partner sponsors were added with that tier